www/caddy/src/opnsense/mvc/app/controllers/OPNsense/Caddy/ReverseProxyController.php

```diff
@@ -38,11 +38,49 @@ class ReverseProxyController extends IndexController
     public function indexAction()
     {
         $this->view->pick('OPNsense/Caddy/reverse_proxy');
-        $this->view->formDialogReverseProxy = $this->getForm("dialogReverseProxy");
-        $this->view->formDialogSubdomain = $this->getForm("dialogSubdomain");
-        $this->view->formDialogHandle = $this->getForm("dialogHandle");
-        $this->view->formDialogAccessList = $this->getForm("dialogAccessList");
-        $this->view->formDialogBasicAuth = $this->getForm("dialogBasicAuth");
-        $this->view->formDialogHeader = $this->getForm("dialogHeader");
+
+        $formDialogReverseProxy = $this->getForm("dialogReverseProxy");
+        $this->view->formDialogReverseProxy = $this->preprocessFields($formDialogReverseProxy, 'reverse.');
+
+        $formDialogSubdomain = $this->getForm("dialogSubdomain");
+        $this->view->formDialogSubdomain = $this->preprocessFields($formDialogSubdomain, 'subdomain.');
+
+        $formDialogHandle = $this->getForm("dialogHandle");
+        $this->view->formDialogHandle = $this->preprocessFields($formDialogHandle, 'handle.');
+
+        $formDialogAccessList = $this->getForm("dialogAccessList");
+        $this->view->formDialogAccessList = $this->preprocessFields($formDialogAccessList, 'accesslist.');
+
+        $formDialogBasicAuth = $this->getForm("dialogBasicAuth");
+        $this->view->formDialogBasicAuth = $this->preprocessFields($formDialogBasicAuth, 'basicauth.');
+
+        $formDialogHeader = $this->getForm("dialogHeader");
+        $this->view->formDialogHeader = $this->preprocessFields($formDialogHeader, 'header.');
+    }
+
+    /**
+     * Preprocess fields to add 'column_id' by stripping prefixes and sort by sequence
+     *
+     * @param array $fields The fields array to process
+     * @param string $prefixToRemove The prefix to strip from 'id'
+     * @return array The processed fields array
+     */
+    private function preprocessFields($fields, $prefixToRemove)
+    {
+        // Add 'column_id' by stripping the prefix
+        foreach ($fields as &$field) {
+            if (isset($field['id'])) {
+                $field['column_id'] = strpos($field['id'], $prefixToRemove) === 0
+                    ? substr($field['id'], strlen($prefixToRemove))
+                    : $field['id'];
+            }
+        }
+
+        // Sort fields by 'sequence', defaulting to 0 if not set
+        usort($fields, function ($a, $b) {
```

Comment:
This here has the side effect to also change the order in the form dialogue. If there should be a /different/ sequence for the columns and the form this solution has to be changed.

Reply:
When I tried to do it like this in the `.volt` template it always crashed. So there are very different possibilities between volt and jinja 2 it seems and I can not sort the same way in the template as e.g.: here:

https://github.com/opnsense/plugins/blob/dbe996b0539d9e8b4627354527f1281cc97aef5d/www/caddy/src/opnsense/service/templates/OPNsense/Caddy/includeLayer4#L15-L23